pkgs/development/ocaml-modules/pipebang/default.nix

```diff
@@ -15,7 +15,7 @@ buildOcaml rec {
 
   strictDeps = true;
 
-  buildInputs = [ camlp4 ];
+  propagatedBuildInputs = [ camlp4 ];
```

Comment:
Did you consider dropping this package altogether?

Reply:
Dropping pipebang?

Reply:
Should I just revert this and then create a separate PR dropping pipebang? I feel like I don't want to bundle it into this PR.

Reply:
Or, we could keep it and open a separate PR either way